# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cramfsswap/fix_makefile.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2006-07-09 15:23:18.000000000 +0000
+++ ./Makefile	2006-07-09 18:49:41.000000000 +0000
@@ -1,10 +1,12 @@
 #!/usr/bin/make -f
+CC=gcc
+STRIP=strip
 
 all: cramfsswap
 
 cramfsswap: cramfsswap.c
-	gcc -Wall -g -O -o cramfsswap -lz cramfsswap.c
-	strip cramfsswap
+	$(CC) -Wall -g -O -o cramfsswap cramfsswap.c -lz
+	$(STRIP) cramfsswap
 
 install: cramfsswap
 	install cramfsswap $(DESTDIR)/usr/bin
